src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -111,54 +134,15 @@ To change the hardware configuration after the cluster is created, see [Manage a
-
-## Step 7. Name the cluster
-
-The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes). A cluster's name cannot be edited after it is created.
-
-Click **Next**. Optionally, you can enable VPC peering for a cluster deployed on GCP. For clusters deployed on AWS, you can [set up AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) after creating your cluster.
-
-## Step 8. Enable VPC Peering (optional)
-
-You can use [VPC peering]({% link cockroachcloud/network-authorization.md %}#vpc-peering) to connect a GCP application to a CockroachDB {{ site.data.products.cloud }} cluster deployed on GCP. A separate VPC Peering connection is required for each cluster.
-
-VPC peering is only available for GCP clusters. For clusters deployed on AWS, you can [set up AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) after creating your cluster. [Azure Virtual Network Peering](https://learn.microsoft.com/azure/virtual-network/virtual-network-peering-overview) is not yet supported. Refer to [CockroachDB {{ site.data.products.dedicated }} on Azure]({% link cockroachcloud/cockroachdb-dedicated-on-azure.md %}).
-
-To continue without enabling VPC peering, click **Next** to [enter billing details](#step-9-enter-billing-details).
-CockroachDB {{ site.data.products.cloud }}
-Otherwise, to enable VPC peering:
-
-1. Under **Additional Settings**, toggle the VPC Peering switch to **Yes**.
-1. Configure the IP address range and size (in CIDR format) for the CockroachDB {{ site.data.products.cloud }} network based on the following considerations:
-      -  As per [GCP's overlapping subnets restriction](https://cloud.google.com/vpc/docs/vpc-peering#restrictions), configure an IP range that doesn't overlap with the IP ranges in your application network.
-      - The IP range and size cannot be changed after the cluster is created. Configuring a smaller IP range size may limit your ability to expand into multiple regions in the future. We recommend configuring an IP range size of `/16` or lower.
-
-        Alternatively, you can use CockroachDB {{ site.data.products.cloud }}'s default IP range and size (`172.28.0.0/14`) as long as it doesn't overlap with the IP ranges in your network.
 
-        To use the default IP range, select **Use the default IP range**. To configure your own IP range, select **Configure the IP range** and enter the IP range and size in CIDR format.
+Click **Next: Finalize**.
 
-        {{site.data.alerts.callout_info}}
-        Custom IP ranges are temporarily unavailable for multi-region clusters.
-        {{site.data.alerts.end}}
+## Step 6. Enter billing details
```

Comment:
Would it be good to mention that you need to be an org admin or have billing coordinator permissions for this step?

Reply:
@jaiayu What if they don't have the permissions? What does the UX for them? What guidance should we provide?